packages/tpetra/core/test/FECrsGraph/FECrsGraph_UnitTests.cpp

```diff
@@ -118,6 +120,10 @@ class GraphPack {
   RCP<const Tpetra::Map<LO,GO,Node> > overlapMap;
   std::vector<std::vector<GO> > element2node;
 
+  // NOTE: This is hardwired for 1D bar elements
+  typedef Kokkos::View<LO*[2], Kokkos::LayoutLeft, typename Node::device_type > k_element2node_type;
```

Comment:
1. It would be nicer to get `device_type` from `Tpetra::Map<LO, GO, Node>::device_type`.  That would minimize changes later on, once we deprecate and remove Node (see https://github.com/trilinos/Trilinos/issues/57).
2. Prefer `using k_element2node_type = ...` C++11 syntax.